schema.json

```diff
@@ -0,0 +1,514 @@
+        "$type": {
+          "const": "LivingDocumentation.Switch, LivingDocumentation.Statements"
+        },
+        "Expression": {
+          "type": "string"
+        },
+        "Sections": {
+          "type": "array",
+          "items": {
+            "Labels": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "Statements": {
+              "$ref": "#/$defs/statements"
+            }
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Expression"
+      ]
+    },
+    "invocation": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.InvocationDescription, LivingDocumentation.Descriptions"
+        },
+        "ContainingType": {
+          "type": "string"
+        },
+        "Name": {
+          "type": "string"
+        },
+        "Arguments": {
+          "$ref": "#/$defs/arguments"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "ContainingType",
+        "Name"
+      ]
+    },
+    "assignment": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.AssignmentDescription, LivingDocumentation.Descriptions"
+        },
+        "Left": {
+          "type": "string"
+        },
+        "Operator": {
+          "type": "string"
+        },
+        "Right": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Left",
+        "Right",
+        "Operator"
+      ]
+    },
+    "return": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.ReturnDescription, LivingDocumentation.Descriptions"
+        },
+        "Expression": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": []
+    },
+    "fields": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "Name": {
+            "type": "string"
+          },
+          "Type": {
+            "type": "string"
+          },
+          "Initializer": {
+            "type": "string"
+          },
+          "Modifiers": {
+            "type": "number"
+          },
+          "Attributes": {
+            "$ref": "#/$defs/attributes"
+          },
+          "DocumentationComments": {
+            "$ref": "#/$defs/documentationComments"
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "Name",
+          "Type"
+        ]
+      }
+    },
+    "documentationComments": {
+      "type": "object",
+      "properties": {
+        "Example": {
+          "type": "string"
+        },
+        "Exceptions": {
+          "type": "object"
+        },
+        "Params": {
+          "type": "object"
+        },
```

Comment:
It may be neat to specify that this specifically has to be an object with string values, like so:

```
        "Params": {
          "type": "object",
          "patternProperties": {
            ".*": {
              "type": "string"
            }
          }
        },
```

(And then probably also for the other properties here, for consistency.)